asr1k_neutron_l3/plugins/l3/service_plugins/l3_extension_adapter.py

```diff
@@ -36,6 +38,9 @@
 
 LOG = log.getLogger(__name__)
 
+TAG_DEFAULT_ROUTE_OVERWRITE = 'custom-default-route'
+TAG_SKIP_MONITORING = 'skip-monitoring'
```

Comment:
Could be put into the `common/asr1k_constants.py`, maybe we want to use them somewhere else in some point in time?

Reply:
Yes that makes sense.